audit/projects/k8s-artifacts-prod/iam.json

```diff
@@ -40,24 +40,12 @@
       ],
       "role": "roles/errorreporting.user"
     },
-    {
-      "members": [
-        "deleted:serviceAccount:k8s-infra-gcr-auditor@k8s-artifacts-prod.iam.gserviceaccount.com?uid=111422293292441494221"
```

Comment:
this is still https://github.com/kubernetes/k8s.io/issues/1730 needing to get resolved